### ISSUE TITLE
gem install CLI behavior not intuitive with --pre in some cases, and should possibly allow multiple -v args

### ISSUE THREAD

**Issue by garysweaver** [rubygems/rubygems#531]
Usually gem command is great, but noticed some wierdness today:

```
$ gem install rails -v '~> 4.0.0'
ERROR:  Could not find a valid gem 'rails' (~> 4.0.0) in any repository
ERROR:  Possible alternatives: rails
```

Ok, that's understandable- there is no 4.0.x yet where last digit x >= 0.
Starts to get a little less intuitive here:

```
$ gem install rails -v '~> 4.0.0' --pre
ERROR:  Could not find a valid gem 'rails' (~> 4.0.0) in any repository
ERROR:  Possible alternatives: rails
```

This works but is the beta1 a last digit? If so, you could never use this to require 4.0.1:

```
$ gem install rails -v '~> 4.0.0.beta1'
Successfully installed rails-4.0.0.beta1
1 gem installed
```

So, then I try:

```
$ gem install rails -v '>= 4.0.0.beta' -v '< 4.1.0'
Successfully installed rails-4.0.0.beta1
1 gem installed
```

but that isn't working intuitively, because this should fail with error or at least not install:

```
$ gem install rails -v '>= 4.0.0.beta' -v '< 3.0.0'
Successfully installed rails-4.0.0.beta1
1 gem installed
```

Long story short:

Should gem install allow this usage?:

```
$ gem install rails -v '>= 4.0.0.beta' -v '< 4.1.0'
```

And when using `--pre` with "approximate" basically install anything with the version '4.0.0.*'?:

```
$ gem install rails -v '~> 4.0.0' --pre
```

(Note: this is not same as #468.)

I took a look through the code, but it wasn't immediately obvious whether there was a workaround/alternative usage, and didn't see pessimistic multiple constaint examples for command-line gem install usage. Thanks in advance!
